src/api/app/components/request_decision_component.rb

```diff
@@ -14,7 +13,7 @@ def render?
   end
 
   def single_action_request
-    @actions.count == 1
+    @bs_request.bs_request_actions.count == 1
```

Comment:
This should be memoized, otherwise you'll fetch the actions over and over again when this method is called.

```suggestion
    @single_action_request ||= @bs_request.bs_request_actions.count == 1
```

Reply:
It's updated